packages/video_player/video_player/lib/video_player.dart

```diff
@@ -97,7 +97,7 @@ class VideoPlayerValue {
   /// Returns [size.width] / [size.height] when size is non-null, or `1.0.` when
   /// size is null or the aspect ratio would be less than or equal to 0.0.
   double get aspectRatio {
-    if (size == null) {
+    if (size == null || size.width == 0 || size.height == 0) {
```

Comment:
can we test this? Should be easy do have a unit test in dart.

Reply:
Yes, let me see if there's any unit test for the aspectRatio

Reply:
Added unit tests for the aspectRatio getter. There were none before, I tested all the cases.